### DISCUSSION TITLE
Operator on GKE

### DISCUSSION THREAD

**Discussion by fabltd** [emqx/emqx-operator#385] [Q&A] [answered]
Hi 

There seems to be a lack of documents on in the configuration options. I am trying to deploy to GKE on Google Cloud. Currently when deploying everything is just a cluster IP. I would like to expose the dashboard as node port and the MQTT port via a load balancer. What needs to be set it doesn’t appear to be very clear ?

**Comment by Rory-Z** [comment, marked as answer]
You can set `listenerServiceTemplate.spec.type: LoadBalancer` like service resource

**Reply by paulleroyza** [reply]
I also found it helpful to check the CRDs, it'll give you all the options. I know it's low level but you could validate your YAML against this: `kubectl get crd emqxbrokers.apps.emqx.io -o yaml`
